meta-rdk-mtk-bpir4/recipes-core/barton-core/files/matter/ExampleTestDACProvider.cpp

```diff
@@ -0,0 +1,22 @@
+// This is a copy of Barton's BartonTestDacProvider for development. This file is intended to showcase how you might
```

Comment:
Assuming that this would be Apache-licensed with Comcast copyright from BartonCore, so at least a two liner for "Copyright 2025 Comcast.../Licensed under the Apache...."?